### PR TITLE
Using Strict parameter within read_csv module

### PR DESCRIPTION
**Pull request by theputta**
##### SUMMARY
In this playbook, the module will raise an exception if it encounters any issues with the CSV input. If you set strict: false, the module will attempt to handle any issues with the CSV input without raising an exception.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
read_csv module

### CODE REVIEW

File: plugins/modules/read_csv.py [hunk 1]
```diff
@@ -92,6 +92,14 @@
   register: users
   delegate_to: localhost
 
+# Use Strict parameter to handle potential issues with the CSV file.
+- name: Read users from CSV file with strict checking
```
Comment (felixfontein) [inline plugins/modules/read_csv.py:96]: This example should not be inserted in this place, between two tasks that belong together.

Also I'm not sure what's your motivation to add a random new example with `strict: true` that doesn't have any details what `strict` does, as opposed to simply adding `strict: true` to an existing example.

**Review by felixfontein — COMMENTED** [review]
Thanks for your contribution.

File: plugins/modules/read_csv.py (hunk 1 above)
Comment (theputta) [reply plugins/modules/read_csv.py]: Sorry for the mishap. I have updated as per your recommendations

File: plugins/modules/read_csv.py [hunk 2]
```diff
@@ -110,10 +110,11 @@
 #   dag;500;500
 #   jeroen;501;500
 
-# Read a CSV file without headers
+# Read a CSV file without headers and handle potential issues with the CSV file.
```
Comment (felixfontein) [inline plugins/modules/read_csv.py:113]:
```suggestion
# Read a CSV file without headers and handle potential issues with the CSV file
```

Comment (felixfontein) [reply plugins/modules/read_csv.py:113]: Isn't it more "fail on potential issues with the CSV file"?

File: plugins/modules/read_csv.py [hunk 3]
```diff
@@ -110,10 +110,11 @@
 #   dag;500;500
 #   jeroen;501;500
 
-# Read a CSV file without headers
+# Read a CSV file without headers and handle potential issues with the CSV file
```
Comment (felixfontein) [inline plugins/modules/read_csv.py:113]: To repeat my previous comment:

> Isn't it more "fail on potential issues with the CSV file"?

**Comment by felixfontein** [comment]
needs_info

**Pull request closed without merging**